### PR TITLE
L1T offline DQM changes for muon resolution plots

### PR DESCRIPTION
**Pull request by thomreis**
Some upgrades to the L1T offline DQM muon resolution histograms.

- Proper delta phi calculation with wrap around at pi.
- Relative pT residuals.

### CODE REVIEW

**Comment by thomreis** [comment]
please test

**Comment by thomreis** [comment]
please test

Unrelated generator error.

**Comment by thomreis** [comment]
+1

**Comment by dmitrijus** [comment]
+1

File: DQMOffline/L1Trigger/src/L1TMuonDQMOffline.cc [hunk 1]
```diff
@@ -90,10 +90,10 @@ L1TMuonDQMOffline::EtaRegion MuonGmtPair::etaRegion() const {
 }
 
 double MuonGmtPair::getDeltaVar(const L1TMuonDQMOffline::ResType type) const {
-    if (type == L1TMuonDQMOffline::kResPt)      return gmtPt() - pt();
-    if (type == L1TMuonDQMOffline::kRes1OverPt) return 1/gmtPt() - 1/pt();
-    if (type == L1TMuonDQMOffline::kResQOverPt) return gmtCharge()/gmtPt() - charge()/pt();
-    if (type == L1TMuonDQMOffline::kResPhi)     return gmtPhi() - phi();
+    if (type == L1TMuonDQMOffline::kResPt)      return (gmtPt() - pt()) / pt();
+    if (type == L1TMuonDQMOffline::kRes1OverPt) return (pt() - gmtPt()) / gmtPt(); // (1/gmtPt - 1/pt) / (1/pt)
```
Comment (fabiocos) [inline DQMOffline/L1Trigger/src/L1TMuonDQMOffline.cc]: why this change? The name Res1OverPt does not seem any more true...

Comment (fabiocos) [reply DQMOffline/L1Trigger/src/L1TMuonDQMOffline.cc]: @thomreis I see what you mean in the comment, it does not seem completely straightforward but ok

**Comment by fabiocos** [comment]
+1
